src/runtime/test/host_port_test.cpp

```diff
@@ -166,6 +172,41 @@ TEST(host_port_test, rpc_group_host_port)
     ASSERT_FALSE(g->contains(hp2));
     ASSERT_EQ(0u, g->members().size());
     ASSERT_EQ(invalid_hp, g->leader());
+
+    // operator <
+    host_port hp_grp1;
+    hp_grp1.assign_group("test_group");
+    if (reinterpret_cast<uint64_t>(hp_grp.group_host_port().get()) <
```

Comment:
The relationship between hp_grp and hp_grp1 definite IMO, so the judgement is not needed.

Reply:
The relationship between hp_grp and hp_grp1 not predictable for my test. I think maybe malloc not usd space ordered.